tidb-cloud/migrate-sql-shards.md

```diff
@@ -0,0 +1,556 @@
+```
+
+In this example, the column IDs of `sale_01` and `sale_02` are auto-increment primary keys. Conflicts might occur when you merge sharded tables in the downstream database. Execute the following SQL statement to modify the PRIMARY KEY attribute of the ID column to normal index:
+
+```sql
+mysql> CREATE TABLE `sales` (
+   ->   `id` bigint(20) NOT NULL ,
+   ->   `uid` varchar(40) NOT NULL,
+   ->   `sale_num` bigint DEFAULT NULL,
+   ->   INDEX (`id`),
+   ->   UNIQUE KEY `ind_uid` (`uid`)
+   -> );
+Query OK, 0 rows affected (0.17 sec)
+```
+
+For more information about the solutions to solve such conflicts, see [Remove the PRIMARY KEY attribute from the column](https://docs.pingcap.com/tidb-data-migration/v5.3/shard-merge-best-practices#remove-the-primary-key-attribute-from-the-column).
+
+### Step 4. Configure Amazon S3 access
+
+To configure Amazon S3 access, follow the instructions in [Configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).
+
+This document uses the Amazon S3 as an example. The following example only lists key policy configurations. Replace the Amazon S3 path with your own values.
+
+```yaml
+{
+   "Version": "2012-10-17",
+   "Statement": [
+       {
+           "Sid": "VisualEditor0",
+           "Effect": "Allow",
+           "Action": [
+               "s3:GetObject",
+               "s3:GetObjectVersion"
+           ],
+           "Resource": [
+               "arn:aws:s3:::dumpling-s3/*"
+           ]
+       },
+       {
+           "Sid": "VisualEditor1",
+           "Effect": "Allow",
+           "Action": [
+               "s3:ListBucket",
+               "s3:GetBucketLocation"
+           ],
+
+           "Resource": "arn:aws:s3:::dumpling-s3"
+       }
+   ]
+}
+```
+
+### Step 5. Perform the data import task
+
+After configuring the IAM Role, you can perform the data import task on the [TiDB Cloud console](https://tidbcloud.com/console/clusters).
+
+1. Navigate to the **Active Clusters** page.
+
+2. Find the area of your target cluster and click **Import Data** in the upper-right corner of the area. The **Data Import** page is displayed.
+
+3. On the **Data Import** page, fill in the following information:
+
+    - **Data Format**: select **CSV**.
+    - **Location**: `AWS`
+    - **Bucket URL**: fill in the bucket URL of your source data. You can use the second-level directory corresponding to tables, `s3://dumpling-s3/store/sales` in this example. You can import the data in all MySQL instances that will be merged into `store.sales` in one go.
+    - **Role-ARN**: enter the Role-ARN you obtained.
+    - **Target Cluster**: fill in the **Username** and **Password** fields.
+
+    Confirm the compliance of cross region. Click **Next**.
+
+4. Modify the file patterns and add the table filter rules if needed. If necessary, you can specify a [table filter](/table-filter.md#syntax). If you want to configure multiple filter rules, use `,` to separate the rules.
```

Comment:
@Frank945946 PTAL

Reply:
1 Yes they are the same. Revised this para to avoid confusion.
2 PM confirmed that there's no need to fill in the "Target table" field in File Pattern.